PixaPicker/ViewController.swift

```diff
@@ -75,7 +95,9 @@ class ViewController: UIViewController, UISearchResultsUpdating, UISearchBarDele
     
     func updateSearchResults(for searchController: UISearchController) {
         guard let searchText = searchController.searchBar.text else { return }
-        (PixaBayAPIService.loadPixaBayRequest(withURL: URLExtensions.pixabaySearchURL(withtext: searchText), completion: {
+        currentSearchText = searchText
+        currentPageNumber = 1
+        (PixaBayAPIService.loadPixaBayRequest(withURL: URLExtensions.pixabaySearchURL(withtext: searchText, withPageNumber: currentPageNumber), completion: {
```

Comment:
All this logic is clean and looks good!   Let's move it to a different class shall we?

Reply:
heck yeah ima get started on that tomorrow! thanks rob